api/HTMLObjectElement.json

```diff
@@ -205,7 +205,7 @@
               "version_added": "12"
             },
             "firefox": {
-              "version_added": "1"
+              "version_added": "4"
```

Comment:
http://mdn-bcd-collector.appspot.com/tests/api/HTMLObjectElement looks sensible and none of these things on `HTMLObjectElement` are reflected attributes, so I trust the results.